File: JavaScript/1175.js
var input = require('fs').readFileSync('stdin', 'utf8');

var lines = input.split('\n');
/*Começa aqui */
let vetor = [];
let entrada;

for(let i = 0; i < 20; i++){
    entrada = parseFloat(lines[i]);
    vetor[i] = entrada;
}

vetor = vetor.slice(0).reverse();

vetor.forEach(function(number, index){
    console.log(`N[${index}] = ${number}`)
})

File: C++/1175.cpp
#include<iostream>

using namespace std;

int main(){

    int vetor[20];
    int entrada, auxiliar;

    for(int i = 0; i < 20; i++){
        cin >> entrada;
        vetor[i] = entrada;
    }

    for(int i = 0, j = 19; i < 10; i++, j--){
        auxiliar = vetor[j];
        vetor[j] = vetor[i];
        vetor[i] = auxiliar;
    }

    for(int i = 0; i < 20; i++){
        cout << "N[" << i << "] = " << vetor[i] << endl;
    }

    return 0;
}